src/Features/CSharpTest/Snippets/CSharpProprSnippetProviderTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Snippets;
+
+public sealed class CSharpProprSnippetProviderTests : AbstractCSharpAutoPropertySnippetProviderTests
+{
+    protected override string SnippetIdentifier => "propr";
+
+    protected override string DefaultPropertyBlockText => "{ get; set; }";
+
+    [WorkItem("https://github.com/dotnet/roslyn/issues/75128")]
```

Comment:
I don't see much value in adding `WorkItem` to every test when you implement a feature from scratch. They are useful when fixing bugs or extending the feature to see how it evolved over time and what edge cases were discovered, but having them like that... not so much. Like what you expect to be done when new tests are added? They will technically be related to the feature request, so is the attribute needed to be duped on every test? I don't like that idea. Consider removing them. Other property snippet tests don't have `WorkItem`s all over the place either

Reply:
Not sure what are the practices in Roslyn for new features, happy to remove the `WorkItem` attributes though